src/components/List/List.stories.js

```diff
@@ -1,13 +1,42 @@
-/* @flow */
```

Comment:
I didn't remove it intentionally, and all those below. Might be that I didn't resolve conflicts appropriately when rebasing on master.